service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/azure/database/ControlledAzureDatabaseResource.java

```diff
@@ -247,7 +247,7 @@ public ApiAzureDatabaseResource toApiResource() {
   private ApiAzureDatabaseAttributes toApiAttributes() {
     return new ApiAzureDatabaseAttributes()
         .databaseName(getDatabaseName())
-        .databaseOwner(getDatabaseOwner().toString())
+        .databaseOwner(Objects.toString(getDatabaseOwner(), null))
```

Comment:
So "null" is an acceptable value here?

Reply:
being null is fine, calling toString on null is the problem